files/routes/comments.py

```diff
@@ -332,9 +332,9 @@ def edit_comment(cid, v):
 @auth_required
 def delete_comment(cid, v):
 	c = get_comment(cid, v=v)
-	if c.deleted_utc: abort(409)
+	if c.state_user_deleted_utc: abort(409)
 	if c.author_id != v.id: abort(403)
-	c.deleted_utc = int(time.time())
+	c.state_user_deleted_utc = time.time()
```

Comment:
This still tries to set it to a unix timestamp (and causes a stack trace in testing).
Recommend using `datetime.now(tz=timezone.utc)` (see later comments on tz-aware datetimes).
Necessary imports if not already available: `from datetime import datetime, timezone`

Reply:
Should now be fixed.